predict.py

```diff
@@ -0,0 +1,105 @@
+"""
+Download the weights in ./checkpoints and ImageNet 1K ID to class mappings beforehand
+wget https://s3.amazonaws.com/deep-learning-models/image-models/imagenet_class_index.json -O in_cls_idx.json
```

Comment:
Can we download the class mapping in the code instead of the instructions?

Also, can the models directly be loaded from torch.hub instead of having users download the models manually?

Reply:
Hi @mannatsingh,

Thank you for your comment! Yes the models can be loaded from the hub on the fly, but we recommend that the checkpoints are downloaded and prepared in the cog environment so the inference can be very fast! You can test on the website that only the first time setting up takes time, but consecutive runs will be very speedy because of this. 
Users who try the web demo do not need to download the checkpoints - everything is there! That note there is just for people who wish to implement models themselves for a Replicate demo too :)

Reply:
Understood! I mentioned this because `torch.hub` also uses a cache to avoid re-downloads (default path is in `~/.cache` but that can be overridden with environment variables). But it's not a big deal :)